config/locales/en.yml

```diff
@@ -1349,6 +1350,11 @@ en:
           other_nationality3: *nationality
           iqts_country: Enter the country or territory the trainee is studying in
       models:
+        placements_form:
+          attributes:
+            placement_ids:
+              too_few_placements: Enter at least 2 placements
+              invalid: Some placements are invalid
```

Comment:
:thinking: How would some be invalid?

Reply:
They shouldn't normally be invalid at this stage with validation on the `PlacementForm`. I'm assuming we would re-validate just before commit. So you _could_ have one or more invalid placements.